### PR TITLE
MQTT C Client

### PR DESCRIPTION
**Pull request by marcin-sielski**

### CODE REVIEW

File: mingw-w64-paho.mqtt.c/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+_realname=paho.mqtt.c
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.1.0
+pkgrel=1
+pkgdesc="Eclipse Paho MQTT C client (mingw-w64)"
+arch=('any')
+url='https://eclipse.org/paho/clients/c/'
+license=('LICENSE')
+validpgpkeys=('gpg_KEY')
```
Comment (Alexpux) [inline mingw-w64-paho.mqtt.c/PKGBUILD]: Do it empty or remove

File: mingw-w64-paho.mqtt.c/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake")
+options=('strip' 'staticlibs')
+source=("https://github.com/eclipse/${_realname}/archive/v${pkgver}.tar.gz"
```
Comment (Alexpux) [inline mingw-w64-paho.mqtt.c/PKGBUILD]: download as named file. Like:
${_realname}-${pkgver}::URL

File: mingw-w64-paho.mqtt.c/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      -G'MSYS Makefiles' \
+      -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+      "${extra_config[@]}" \
+      -DBUILD_{SHARED,STATIC}_LIBS=ON \
+      ../${_realname}-${pkgver}
+
+  make
+}
+
+
```
Comment (Alexpux) [inline mingw-w64-paho.mqtt.c/PKGBUILD]: Remove extra line

File: mingw-w64-paho.mqtt.c/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+_realname=paho.mqtt.c
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.1.0
+pkgrel=1
+pkgdesc="Eclipse Paho MQTT C client (mingw-w64)"
+arch=('any')
+url='https://eclipse.org/paho/clients/c/'
+license=('LICENSE')
```
Comment (mati865) [inline mingw-w64-paho.mqtt.c/PKGBUILD]: Fix license

**Pull request merged**